plugins/view-transitions/includes/settings.php

```diff
@@ -72,15 +75,17 @@ function plvt_get_setting_default(): array {
  *
  * @since 1.0.0
  *
- * @return array{ default_transition_animation: non-empty-string, header_selector: non-empty-string, main_selector: non-empty-string, post_title_selector: non-empty-string, post_thumbnail_selector: non-empty-string, post_content_selector: non-empty-string } {
+ * @return array{ default_transition_animation: non-empty-string, default_transition_animation_duration: non-empty-string, header_selector: non-empty-string, main_selector: non-empty-string, post_title_selector: non-empty-string, post_thumbnail_selector: non-empty-string, post_content_selector: non-empty-string } {
  *     Stored setting value.
  *
- *     @type string $default_transition_animation Default view transition animation.
- *     @type string $header_selector              CSS selector for the global header element.
- *     @type string $main_selector                CSS selector for the global main element.
- *     @type string $post_title_selector          CSS selector for the post title element.
- *     @type string $post_thumbnail_selector      CSS selector for the post thumbnail element.
- *     @type string $post_content_selector        CSS selector for the post content element.
+ *     @type string $default_transition_animation          Default view transition animation.
+ *     @type string $default_transition_animation_duration Default transition animation duration in milliseconds.
+ *                                                         Added in 1.x.x.
```

Comment:
```suggestion
 *                                                         Added in n.e.x.t
```

Should be `n.e.x.t` so it will set next plugin version. 🔢 Update in other places also